internal-services/catalog/create-advisory-task.yaml

```diff
@@ -0,0 +1,115 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: task-create-advisory
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+      Pushes an advisory yaml to a Git repository
+  params:
+    - name: repo
+      type: string
+      description: Git repository to push the advisory to
+    - name: advisory_json
+      type: string
+      description: |
+          String containing a JSON representation of the advisory data. It should not contain the `advisory`
+          top level key
+          E.g. '{"spec":{"product_id":123,"type":"RHSA"}}'
+    - name: application
+      type: string
+      description: Application being released
+  results:
+    - name: result
+      description: Success if the task succeeds, the error otherwise
+  steps:
+    - name: create-advisory
+      image: quay.io/jbieren/release-utils:ansible_5
+      env:
+        - name: GITLAB_HOST
+          valueFrom:
+            secretKeyRef:
+              name: create-advisory-secret
+              key: gitlab_host
+        - name: ACCESS_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: create-advisory-secret
+              key: gitlab_access_token
+        - name: GIT_AUTHOR_NAME
+          valueFrom:
+            secretKeyRef:
+              name: create-advisory-secret
+              key: git_author_name
+        - name: GIT_AUTHOR_EMAIL
+          valueFrom:
+            secretKeyRef:
+              name: create-advisory-secret
+              key: git_author_email
+      script: |
+          #!/usr/bin/env sh
+          set -eox pipefail
+
+          REPO_BRANCH=main
+
+          # loading git and gitlab functions
```

Comment:
Yeah, when they were added, I opposed to the idea that one of the functions file would call an init function upon loading (with `source` or `.`). So that was removed and we now call the init function separately. But sourcing the function files in the Dockerfile always would probably be fine.

Reply:
(But I think this is definitely out of scope for this PR. @davidmogar if you feel this should be changed, feel free to create a story for it.)